wordpress-scripts-webpack.config.js

```diff
@@ -65,6 +66,18 @@ module.exports = {
         adminBlocks: path.resolve(process.cwd(), 'blocks', 'load.js'),
         campaignBlocks: srcPath('Campaigns/Blocks/blocks.ts'),
     },
+    plugins: [
+        ...defaultConfig.plugins,
+        new CopyWebpackPlugin({
```

Comment:
What's happening here?

Reply:
Copying every `style.css` and `script.js` files found in the blocks folders to their respective build folder.